### DISCUSSION TITLE
Meaning of hits

### DISCUSSION THREAD

**Discussion by andrusito** [YOURLS/YOURLS#2958] [Q&A] [answered]
Hi! I'm using this awesome script to redirect several old domains to a new domain.

Every year I look at stats to determine if these old domains are currently bringing significative traffic, and if they are not I can go ahead and not renew them.

I do this by calculating the amount of traffic I get and the page RPM.

But when I go to YOURLS' stats it says something like "Last 30 days: 594 hits (19.8 per day)". I looked around and I could not find an answer, so I need to know what exactly are hits, because I'd like to count only real visitors, and not bots... is that possible?

**Comment by ozh** [comment, marked as answer]
one hit = one client (your firefox, someone's chrome, googlebot, ) requesting a short URL (`http://sho.rt/blah`) that has not been previously stored in its cache.

You go to https://yourls.org/f1 : it will count as one hit, and you will be redirected here.
Go to https://yourls.org/f1 again : your browser has the redirection in its cache, so it will go directly to this page and there won't be a new hit.

**Reply by andrusito** [reply]
great! thanks a lot @ozh, is there a way to filter bot hits so they don't get counted? or to put it differently, is there a way to count hits from real users?

**Reply by ozh** [reply]
there are plugins for this

**Reply by andrusito** [reply]
thanks, @ozh - I have these plugins related to what I'm trying to accomplish installed:

Don't Log Bots
Don't Log Crawlers 
Don't Log Health Checker

However, when I check "traffic sources / referrers" I see lots of these type of links:

http://www.mysite.net/blog/images/radio_sidebar.png: 499 - what does it mean?

**Reply by ozh** [reply]
I don't know

**Comment by andrusito** [comment]
Hi, @ozh! Another related question: when I check "traffic sources / referrers" I see lots of these type of links:

http://www.mysite.net/blog/images/radio_sidebar.png: 499 - what does it mean?
